fdbserver/DataDistribution.actor.cpp

```diff
@@ -2932,11 +2945,13 @@ ACTOR Future<Void> teamTracker(DDTeamCollection* self, Reference<TCTeamInfo> tea
 				}
 
 				lastZeroHealthy = self->zeroHealthyTeams->get(); //set this again in case it changed from this teams health changing
-				if( self->initialFailureReactionDelay.isReady() && !self->zeroHealthyTeams->get() ) {
+				if ((self->initialFailureReactionDelay.isReady() && !self->zeroHealthyTeams->get()) ||
+				    teamContainsFailedServer(self, team)) {
```

Comment:
`teamContainsFailedServer` is an expensive operation.
We may want to assign it to a variable outside the `if` and also reuse it in the following appearance of `teamContainsFailedServer`